js/__internal/m_draggable.ts

```diff
@@ -710,24 +725,46 @@ const Draggable = (DOMComponent as any).inherit({
     return $(container);
   },
 
-  _dragMoveHandler(e, scrollBy) {
+  _getDraggableElementOffset(initialOffsetX: number, initialOffsetY: number): Offset {
+    const initScrollTop = this._initScrollTop ?? 0;
+    const initScrollLeft = this._initScrollLeft ?? 0;
+
+    const scrollTop = this._getScrollableScrollTop();
+    const scrollLeft = this._getScrollableScrollLeft();
+
+    const isFixedPosition = ($(this.element()) as any).css('position') === 'fixed';
+
+    const result: Offset = {
+      left: (this._startPosition?.left ?? 0) + initialOffsetX,
+      top: (this._startPosition?.top ?? 0) + initialOffsetY,
+    };
+
+    if (!isFixedPosition && !this._hasClonedDraggable()) {
+      if (isNumeric(scrollTop)) {
+        result.top += scrollTop - initScrollTop;
+      }
+      if (isNumeric(scrollLeft)) {
+        result.left += scrollLeft - initScrollLeft;
+      }
+    }
+
+    return result;
+  },
```

Comment:
Let's make it shorter:

```suggestion
    const result: Offset = {
      left: (this._startPosition?.left ?? 0) + initialOffsetX,
      top: (this._startPosition?.top ?? 0) + initialOffsetY,
    };
    
    if (isFixedPosition || this._hasClonedDraggable()) {
      return result;
    }
    
    return {
      left: isNumeric(scrollLeft)
        ? result.left + scrollLeft - initScrollLeft
        : result.left,
      top: isNumeric(scrollTop) 
        ? result.top + scrollTop - initScrollTop
        : result.top,
    }
  },
```